### PR TITLE
Add shopper assistant script

### PR DESCRIPTION
**Pull request by paulhowardflanders**
# Description
Adds shopper assistant script to M2 plugin. I tested this change on my M2 devbox running against a merchant in staging.

#changelog Add shopper assistant script

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [x] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# For PR Reviewer 

- [ ] Reviewed unit tests to make sure we are using real components rather than mocks as much as possible?
- [ ] For any major change (observer, new Bolt feature, core Magento interaction) we must add a feature switch, did you verify this?

# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [ ] I have added my ticket link and provided a changelog message.

https://user-images.githubusercontent.com/55464043/179085992-fedc5c20-27ee-4b64-8a56-da343401fd8f.mov

### CODE REVIEW

File: view/frontend/templates/js/boltjs.phtml [hunk 1]
```diff
@@ -90,3 +91,12 @@ if ($isLoadBoltJs && $isLoadConnectJs) {
         data-publishable-key="<?= /* @noEscape */ $checkoutKey; ?>">
     </script>
 <?php endif; ?>
+<?php if ($isShopperAssistantEnabledJs): ?>
```
Comment (vitaliyreznikov) [inline view/frontend/templates/js/boltjs.phtml:94]: We want to have this script in all pages, right?

**Review by vitaliyreznikov — APPROVED** [review]
LGTM, just couple questions

File: view/frontend/templates/js/boltjs.phtml (hunk 1 above)
Comment (paulhowardflanders) [reply view/frontend/templates/js/boltjs.phtml:94]: Yup!

File: Helper/FeatureSwitch/Definitions.php [hunk 2]
```diff
@@ -587,6 +592,12 @@ class Definitions
             self::VAL_KEY         => true,
             self::DEFAULT_VAL_KEY => false,
             self::ROLLOUT_KEY     => 100
+        ],
+        self::M2_ENABLE_SHOPPER_ASSISTANT => [
+            self::NAME_KEY        => self::M2_ENABLE_SHOPPER_ASSISTANT,
+            self::VAL_KEY         => false,
```
[the remark this answers is not in the document]
Comment (paulhowardflanders) [reply Helper/FeatureSwitch/Definitions.php]: Gotcha, updated to true/false/0. Thanks! 🙂

**Pull request merged**